build/release.sh

```diff
@@ -33,7 +33,8 @@ function create_release_data()
   "tag_name": "${VERSION}",
   "name": "${VERSION}",
   "draft": true,
-  "prerelease": false
+  "prerelease": false,
+  "body": "https://github.com/phrase/openapi/blob/master/clients/cli/CHANGELOG.md"
```

Comment:
Hi @skyfrk thank you for contributing, could you please open the pull request against [phrase/openapi](https://github.com/phrase/openapi) and also update the URL to point to the changelog of phrase-cli?
```suggestion
  "body": "https://github.com/phrase/phrase-cli/blob/master/CHANGELOG.md"
```

Reply:
Thanks a lot, I will take a look in that one and close this one then.